eth-connector/src/lib.rs

```diff
@@ -253,6 +253,9 @@ impl EthConnectorContract {
 
         this.acl_init_super_admin(env::predecessor_account_id());
         this.acl_grant_role("DAO".to_string(), owner_id.clone());
+        this.acl_grant_role("PauseManager".to_string(), env::predecessor_account_id());
```

Comment:
The DAO role is enough to do pause and unpause

Reply:
Btw, what is the owner_id and why we need to pass it on init?

Reply:
But `env::predecessor_account_id()` doesn't have DAO role

Reply:
If we can init the EthConnector by using DAO we can remove owner_id. But I am not sure, that we can: metadata can be too big to be signed by Ledger.

Also I am not sure, that it is good idea to give the predecessor_account_id the super admin role. 

I need to give to predecessor_account_id DAO or PauseManager role, but I can remove it after pausing....

Reply:
@olga24912 could you a bit clarify why we don't need `owner_id` for DAO?